estuary/api/v1.py

```diff
@@ -281,7 +281,18 @@ def get_siblings(resource, uid):
         serialized_node['resource_type'] = inflated_node.__label__
         serialized_results.append(serialized_node)
 
-    return jsonify(serialized_results)
+    uid_name = story_node.unique_id_property
+    description = get_siblings_description(
+        getattr(story_node, uid_name), story_node_story_flow, backward)
+
+    result = {
+        'data': serialized_results,
+        'meta': {
+            'description': description[0].upper() + description[1:]
```

Comment:
The code making the first letter uppercase should likely be part of `get_siblings_description`.